---a/net/minecraft/enchantment/EnchantmentDigging.java
+++b/net/minecraft/enchantment/EnchantmentDigging.java
@@ -4,34 +4,29 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.ResourceLocation;
 
-public class EnchantmentDigging extends Enchantment
-{
+public class EnchantmentDigging extends Enchantment {
+
     private static final String __OBFID = "CL_00000104";
 
-    protected EnchantmentDigging(int enchID, ResourceLocation enchName, int enchWeight)
-    {
+    protected EnchantmentDigging(int enchID, ResourceLocation enchName, int enchWeight) {
         super(enchID, enchName, enchWeight, EnumEnchantmentType.DIGGER);
         this.setName("digging");
     }
 
-    public int getMinEnchantability(int enchantmentLevel)
-    {
+    public int getMinEnchantability(int enchantmentLevel) {
         return 1 + 10 * (enchantmentLevel - 1);
     }
 
-    public int getMaxEnchantability(int enchantmentLevel)
-    {
+    public int getMaxEnchantability(int enchantmentLevel) {
         return super.getMinEnchantability(enchantmentLevel) + 50;
     }
 
-    public int getMaxLevel()
-    {
+    public int getMaxLevel() {
         return 5;
     }
 
-    public boolean canApply(ItemStack stack)
-    {
+    public boolean canApply(ItemStack stack) {
         return stack.getItem() == Items.shears ? true : super.canApply(stack);
     }
-}
+}
